docs/release-notes/8.0.32-24.md

```diff
@@ -0,0 +1,70 @@
+# Percona Server for MySQL 8.0.32-24 (2023-03-12)
+
+<style>
+    table {
+        width=50%’;
+        font-family: Poppins;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+</style>
+
+| Release date | {{ release_date }} |
+|--- | --- |
+| Install instructions | [Install Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/installation.html) |
+
+[Percona Server for MySQL 8.0.32-24](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.32 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-32.html)
+in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a free, fully compatible, enhanced, and open
+source drop-in replacement for any MySQL database. It provides superior
+performance, scalability, and instrumentation.
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. It
+delivers more value to MySQL server users with optimized performance,
+greater performance scalability and availability, enhanced backups, and
+increased visibility. 
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+## Release highlights
+
+Percona decided to revert the following MySQL bug fix:
+
+The data and the GTIDs backed up by mysqldump were inconsistent when the options --single-transaction and --set-gtid-purged=ON were both used. It was because in between the transaction started by mysqldump and the fetching of GTID_EXECUTED, GTIDs on the server could have increased already. With this fixed, a FLUSH TABLES WITH READ LOCK is performed before the fetching of GTID_EXECUTED to ensure its value is consistent with the snapshot taken by mysqldump.
```

Comment:
We can also mention that in comparison to the upstream, Percona's solution does not have the limitation mentioned in the MySQL 8.0.32 release notes
```
Limitation: This fix adds a requirement for the [RELOAD](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_reload) privilege when using [--single-transaction](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_single-transaction) to execute FLUSH TABLES WITH READ LOCK; the MySQL team is investigating a solution. (Bug #33630199, Bug #105761)
```
and recommend Oracle MySQL Server users to use `mysqldump` utility from the Percona Server (if @hrvojem confirms this is safe)

Reply:
Added.

Reply:
Added.